demisto_sdk/commands/common/tools.py

```diff
@@ -411,7 +409,7 @@ def get_marketplace_to_core_packs() -> Dict[MarketplaceVersions, Set[str]]:
     Returns:
         A mapping from marketplace versions to their core packs.
     """
-    if is_external_repository():
+    if tools_core.is_external_repository():
```

Comment:
why add this prefix anywhere, rather than of importing `is_external_repository` (and all others) as we did before? much more readable imo

Reply:
https://github.com/demisto/demisto-sdk/pull/3324/commits/16ac8a9c40aef53367677805d92c8733ad2a3270